cmd/loadgenerator/main.go

```diff
@@ -36,6 +37,12 @@ import (
 
 const transactionBlockSize = 800
 
+var runOnce = flag.Bool("once", false, "Terminate after first spend loop")
+
+func init() {
+	flag.Parse()
```

Comment:
I would have put this in the first line of `main`, but your implementation is not wrong.